wallet/src/wallet/tests.rs

```diff
@@ -681,7 +681,7 @@ fn issue_and_transfer_tokens(#[case] seed: Seed) {
 
     let amount_fraction = (block1_amount.into_atoms() - NETWORK_FEE) / 10;
 
-    let token_amount_to_issue = Amount::from_atoms(rng.gen_range(1..amount_fraction));
+    let token_amount_to_issue = Amount::from_atoms(rng.gen_range(2..amount_fraction));
```

Comment:
🤦 same bug as before, I thought I fixed it, apparently not. 
the actual bug is on line 736,
`Amount::from_atoms(rng.gen_range(1..token_amount_to_issue.into_atoms()));`
you can change it to 
`Amount::from_atoms(rng.gen_range(1..=token_amount_to_issue.into_atoms()));`
but then you will also need to handle the case when all of the tokens are transferred so line 778 will be empty, as the balance will be 0.

Reply:
I guess assert for <= 1 or remove the assert altogether?, and then get first() or Amount::ZERO

Reply:
Went with these changes.